database/dbProgramInterestForm.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+function createProgramInterestForm($form) {
+    $connection = connect();
+
+    $family_id = $_SESSION['_id'];
+    $first_name = $form["first_name"];
+    $last_name = $form["last_name"];
+    $address = $form["address"];
+	$city = $form["city"];
+    $neighborhood = $form["neighborhood"];
+    $state = $form["state"];
+    $zip = $form["zip"];
+    $cell_phone = $form["cell_phone"];
+    $home_phone = $form["home_phone"];
+    $email = $form["email"];
+    $child_num = $form["child_num"];
+    $child_ages = $form["child_ages"];
+    $adult_num = $form["adult_num"];
+	$id = null;
+
+    // Begin transaction so if there are any errors with any queries, they are all rolled back and no data is inserted into database
+    // This is done so if one query is succesful and another isn't, it doesn't just insert the successful one which would result in incomplete data in the database
+    mysqli_begin_transaction($connection);
+    try {
+        // First query inserts basic information in dbProgramInterestForm
+        $query = "
+            INSERT INTO dbProgramInterestForm (family_id, first_name, last_name, address, city, neighborhood, state, zip, cell_phone, home_phone, email, child_num, 
+            child_ages, adult_num)
+            values ('$family_id', '$first_name', '$last_name', '$address', '$city', '$neighborhood', '$state', '$zip', '$cell_phone', '$home_phone', '$email', '$child_num', 
+            '$child_ages', '$adult_num');
+        ";
+        $result = mysqli_query($connection, $query);
+        if (!$result) {
+            return null;
+        }
+        $id = mysqli_insert_id($connection);
+
+        // Insert program interests
+        if (isset($form['programs'])) {
+            $programs = $form['programs'];
+            insertProgramInterests($programs, $id, $connection);
```

Comment:
you could add a statement to check if the return from this function is null to know if the query was successful in this function